src/scripts.js

```diff
@@ -84,7 +91,25 @@ function getDate() {
   return myDate;
 };
 
-function resetHydrationData() {
+function resetWidgetData() {
```

Comment:
still wondering how we can make this more dynamic.. we were going to add a parameter for which widget is being passed but got hung up on how exactly to pass in that information.  Maybe we could try passing our global query selectors?

Reply:
I feel like we made it better with the changes, but not necessarily more dynamic. I agree that we need to keep working on it.